### ISSUE TITLE
[COGNITION]: GIBCT® - Consider adding the semantic lists for tuition and housing breakdowns

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#13761]
## Description

Our tuition and housing cost lists are marked up with non-semantic divs right now. I think it would be beneficial to make them into headings and heading-labeled lists for screen reader context. Screenshot attached below.

## Related Issue
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/1277

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] The visual layout should not change at mobile, mid-width, or large screen layouts
- [ ] There must be zero aXe violations when scanned with browser plugin or CI/CD tests.
- [ ] HTML must validate against the W3C HTML5 validator tool. Invalid markup makes this pattern troublesome.
- [ ] Code is implemented exactly as offered in the HTML snippet. Otherwise screen readers will ignore the list item semantics.

## Required Markup Change
```html
<h4 id="tuition-and-fees">Tuition and fees</h4>
<a href="#">Learn more</a>
<ul aria-labelledby="tuition-and-fees" role="list">
  <li>
    <span>Fall:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>Spring:</span>
    <span>$3,548</span>
  </li>
  <li>
    <span>
      <b>Total per year:</b>
    </span>
    <span>$7,096</span>
  </li>
</ul>

...
<!-- Same pattern for the housing allowance and book stipend groups -->
```

## Screenshots or Trace Logs

![Screen Shot 2020-09-18 at 4 54 44 PM](https://user-images.githubusercontent.com/934879/93648364-3b340480-f9d0-11ea-8ef1-6fb7c78ee9f8.png)

**Comment by hughes-dustin** [comment]
PR: https://github.com/department-of-veterans-affairs/vets-website/pull/14382
branch: bah-13761

**Comment by delli-gatti-michael** [comment]
Local Testing complete

**Comment by delli-gatti-michael** [comment]
@1Copenut This fix is on staging and ready for review

**Comment by 1Copenut** [comment]
Looks 💯 Moving to close.
